saleor/payment/gateways/np_atobarai/__init__.py

```diff
@@ -18,28 +21,22 @@
 logger = logging.getLogger(__name__)
 
 
-def inject_api_config(fun):
-    def inner(payment_information: PaymentData, config: GatewayConfig):
-        return fun(payment_information, get_api_config(config.connection_params))
-
-    return inner
-
-
 def parse_errors(errors: List[str]) -> str:
-    # FIXME: better solution?
-    #  Transaction.error in database has max_length of 256
-    return os.linesep.join(errors)[:256]
+    return os.linesep.join(errors[:11])
```

Comment:
Could we add a comment here? Previously I could understand the slice and the db constraint and accept the fact that 256 characters will be enough to debug stuff. In the current implementation 11 looks like a magic number